front-packages/akeneo-design-system/src/components/Tags/Tags.tsx

```diff
@@ -0,0 +1,107 @@
+import React, {Ref, ReactNode, isValidElement, ReactElement} from 'react';
+import styled from 'styled-components';
+import {AkeneoThemedProps} from 'theme';
+
+/**
+ * The colors defined in this file are the alternative ones
+ * You will find the hex colors on this page:
+ * https://www.notion.so/akeneo/Alternative-colors-0f5283c1b02f4fd4a418f1e20f2efa99
+ * Those colors will most likely only be used with the tags components
+ */
+const Tag = styled.li<
+  {tint: 'green' | 'blue' | 'dark_blue' | 'purple' | 'dark_purple' | 'yellow' | 'red'} & AkeneoThemedProps
```

Comment:
```suggestion
type Tint = 'green' | 'blue' | 'dark_blue' | 'purple' | 'dark_purple' | 'yellow' | 'red';
type TagProps = {
  tint: Tint;
} & React.HTMLAttributes<HTMLLIElement>;
const Tag = styled.li<TagProps & AkeneoThemedProps>`;
```
Extending the `HTMLAttributes<HTMLLIElement>` will allow you to get the `title` prop you use later 😉